### ISSUE TITLE
More MassGIS files

### ISSUE THREAD

**Issue by jleila** [kseignon/Mini-project-3#4]
According to the R Package help section under "Macleish" > "MassGIS", this is how we can download MassGIS shapefiles: 
mass_gis(layer = "contours250k")

This is one way to find relationships between MassGIS and MacLeish Data.
macleish_intersect(sp)

I am trying this and also  annotating the code we have and outlining the remaining point of the project which I will post in another issue.

**Comment by kseignon** [comment]
Yes! Julia, we have to intersect the boundary data with the contour data to restrict the contour data to Macleish
